### ISSUE TITLE
Pagination directive issue

### ISSUE THREAD

**Issue by nkamenar** [michaelbromley/angularUtils#321]
version : "0.0.1"

Description of Issue: I am getting the following in my console whenever the page loads:
Pagination directive: the pagination controls cannot be used without the corresponding pagination directive, which was not found at link time.
In addition the pagination seems to work as intended with the exception that the paging controller just looks like a bulleted list instead of the nice button graphics like you have in your examples. I do have bootstrap on the page for the record.

Expected result: No console errors logged and pagination control should show properly styled with bootstrap

Any relevant code:
```
Here is my table row:
<tr dir-paginate="item in postItemsDisplay | itemsPerPage: listItemsPerPage" total-items="totalItems" current-page="pagination.current">
    <td style="width: 50px;">
        <button type="button" class="btn btn-default btn-tny" data-ng-click="CopySingleItem(item.Id)" data-toggle="modal" data-target="#modal2">Edit</button></td>
    <td><a title="View Item" onmouseover="javascript:this.style.cursor='pointer';" data-ng-click="SPModal(item.Id)">{{item.Title}}</a></td>
    <td>{{item.SingleText1}}</td>
    <td>{{item.SingleText2}}</td>
    <td>{{item.ChoiceColumnValue}}</td>
    <td style="width: 30px;">
        <button type="button" class="btn btn-danger btn-tny" data-ng-click="DeleteConfirmation('TestList',item.Id)" data-toggle="modal" data-target="#deleteConfirmation">&times;</button></td>
</tr>

and here is my pagination control
<dir-pagination-controls on-page-change="pageChanged(newPageNumber)"></dir-pagination-controls>
```
It ends up looking like the following img and the links to the pages work but I don't know why they aren't styled:
![screen shot 2016-03-24 at 11 08 34 am](https://cloud.githubusercontent.com/assets/8999291/14021201/e20b507e-f1b0-11e5-95bd-890ff12fd5fc.png)

**Comment by michaelbromley** [comment]
Hi,

Are you able to reproduce this in a Plunker demo? I just created a bare-bones template you can fork: http://plnkr.co/edit/b37IdFFJUokaeSummETX?p=preview

**Comment by nkamenar** [comment]
No I haven't been able to replicate the error on Plunker. I am using this on a sharepoint site with asynchronous $http calls to fetch data from sharepoint lists and I can't replicate that within Plunker. I have managed to make progress though. I have been manually applying the bootstrap paginate styles but I can't seem to get it quite right. Here is the CSS I have added thus far:

`.pagination > li > a,
.pagination > li > span {
    position: relative;
    float: left;
    padding: 6px 12px;
    margin-left: -1px;
    line-height: 1.42857143;
    color: #428bca;
    text-decoration: none;
    background-color: #fff;
    border: 1px solid #ddd;
}

.pagination > li > a:hover,
.pagination > li > span:hover,
.pagination > li > a:focus,
.pagination > li > span:focus {
    color: #2a6496;
    background-color: #eee;
    border-color: #ddd;
}

.pagination > .active > a,
.pagination > .active > span,
.pagination > .active > a:hover,
.pagination > .active > span:hover,
.pagination > .active > a:focus,
.pagination > .active > span:focus {
    z-index: 2;
    color: #fff;
    cursor: default;
    background-color: #428bca;
    border-color: #428bca;
}`

This is mostly working but some borders aren't showing right and the active page doesn't highlight right when I don't have the mouse over it. If I hover the mouse over it it looks like this:
![screen shot 2016-03-24 at 1 46 42 pm](https://cloud.githubusercontent.com/assets/8999291/14026011/28d47e34-f1c7-11e5-8f09-6c7fa7b375c9.png)

But if I move my mouse away it goes to this like something is overlapping it or something.
![screen shot 2016-03-24 at 1 46 25 pm](https://cloud.githubusercontent.com/assets/8999291/14026022/3a2d25f0-f1c7-11e5-9606-2c19c729120b.png)
